src/mem/ruby/system/VIPERSequencer.cc

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2024 UW Madison
+ *
```

Comment:
These should be "Copyright (c) 2024 The University of Wisconsin"

Reply:
@mjkpolo just making sure you saw this

Reply:
ope I forgot about this let me make that change quick